src/rules/color-hex-notation/__tests__/index.js

```diff
@@ -0,0 +1,43 @@
+import {
+  ruleTester,
+  warningFreeBasics
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule("lowercase", tr => {
+  warningFreeBasics(tr)
+
+  tr.ok("a { color: pink; }")
+  tr.ok("a { color: #000; }")
+  tr.ok("a { something: #000, #fff, #ababab; }")
+  tr.ok("a { color: #0000ffcc; }", "eight digits")
+  tr.ok("a { color: #00fc; }", "four digits")
+
+  tr.ok("a { padding: 000; }")
+  tr.ok("a::before { content: \"#ababa\"; }")
```

Comment:
You may also want to put a similar check in each `testRule()` instance to ensure that it ignores comments (theoretically should because of `styleSearch`, but just being thorough I guess).